Lib/tarfile.py

```diff
@@ -1875,7 +1875,7 @@ def list(self, verbose=True, *, members=None):
         """
         self._check()
 
-        if members is None:
+        if members is None or members == []:  # None or empty list
```

Comment:
I think PEP 8 recommends to not add comments that just duplicate the code.  This one says what the code is doing, but not why it is desirable.

Reply:
The two sets of changes are largely duplicates and try to make the command lines equivalent between the two.
Good point on the comment, changed to why needed.
Spell Checked and a couple of minor changes to the News entry is that the doc that you had in mind?